adivinar_numero.rb

```diff
@@ -0,0 +1,11 @@
+n = rand(1..10)
+puts "Bienvenido al programa, Adivina el número que estoy pensando"
+puts "Por favor ingresa el número"
+num = gets.chomp.to_i
+while num!=n
+    puts "Intenta de nuevo. "
+    puts "Por favor ingresa el número"
+    num = gets.chomp.to_i
```

Comment:
for this particular case the variable would be valid but keep in mind that it is a good practice to use telling names.
```suggestion
    number = gets.chomp.to_i
```

Reply:
ruby es un lenguaje muy diciente, por lo que importa mucho que el nombre de cada variable o clase o servicios sea congruente a lo que va  a hacer o almacenar, en este caso como almacena un numero, el nombre de la variable number queda mejor que num, de tal modo que si otro dev la ve sabe que guarda